### PR TITLE
5747 - Existing Request Search Screen

### PR DESCRIPTION
**Pull request by scottrumsby**
*Issue #:* /bcgov/entity#5747

*Description of changes:*

- Made the changes from the first slide of existing request search design document

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
According to the InVision, the Retrieve Name Request button should stay blue and should activate validation on the required fields.
However, when I land on this page, this button is disabled.

**Comment by severinbeauvais** [comment]
1. The Invision specs out a gutter of ~35px~ 40px around everything. However, there is extra left/right whitespace around the content. Please remove it.

2. The top text needs to be updated. (You can leave the : at the end.)

3. See comment above regarding the Retrieve button. I confirmed with Tracey that this button should always be enabled (as it is not AA compliant when disabled). Clicking it should show any invalid/incomplete fields on the page (and obviously not submit if invalid).

**Review by severinbeauvais — CHANGES_REQUESTED** [review]
See comments above (which Tracey and I went through together).

File: client/src/components/existing-request/existing-request-search.vue [hunk 1]
```diff
@@ -10,50 +10,53 @@
...
-                       @input="setExistingRequestSearch('nrNum', $event)"
-                       filled
-                       validate-on-blur
-                       :rules="nrRules"
-                       placeholder="Business Name or NR Number"
-                       id="nr-num-text-field" />
+                      @input="setExistingRequestSearch('nrNum', $event)"
+                      class="copy-normal"
+                      filled
+                      validate-on-blur
```
Comment (cameron-freshworks) [inline client/src/components/existing-request/existing-request-search.vue:23]: What does this prop do? Sounds fancy!

**Review by cameron-freshworks — COMMENTED** [review]
From a code standpoint it looks good, tho I think Severin and Tracey had some UX/design tweaks to be made 👍

File: client/src/components/existing-request/existing-request-search.vue (hunk 1 above)
Comment (scottrumsby) [reply client/src/components/existing-request/existing-request-search.vue:23]: validate-on-blur?  it prevents the field validation from happening until after the field is removed from the focus so the red error messages don't show up from the first character entered (useful if the field has a length rule for instance)

Comment (cameron-freshworks) [reply client/src/components/existing-request/existing-request-search.vue:23]: Ah, very cool! Thanks sir

**Review by cameron-freshworks — APPROVED** [review]

**Comment by severinbeauvais** [comment]
I have verified the changes - looks good, thanks!

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**